crates/bevy_pbr/src/extended_material.rs

```diff
@@ -97,13 +97,20 @@ pub trait MaterialExtension: Asset + AsBindGroup + Clone + Sized {
 /// When used with `StandardMaterial` as the base, all the standard material fields are
 /// present, so the `pbr_fragment` shader functions can be called from the extension shader (see
 /// the `extended_material` example).
-#[derive(Asset, Clone, TypePath)]
+#[derive(Asset, Clone, Reflect)]
+#[reflect(type_path = false)]
 pub struct ExtendedMaterial<B: Material, E: MaterialExtension> {
     pub base: B,
     pub extension: E,
 }
 
-impl<B: Material, E: MaterialExtension> AsBindGroup for ExtendedMaterial<B, E> {
+// We don't use the `TypePath` derive here due to a bug where `#[reflect(type_path = false)]`
+// causes the `TypePath` derive to not generate an implementation.
+impl_type_path!((in bevy_pbr::extended_material) ExtendedMaterial<B: Material, E: MaterialExtension>);
+
+impl<B: Material, E: MaterialExtension> AsBindGroup
```

Comment:
You will need to run `cargo fmt` on this file as it is currently failing CI. Normally I would do this for you, but it looks like I don't have permission to edit this branch (this is an option you can check when you open a PR).

Reply:
Done!